src/ec/packages/ec-component-inpage-navigation/inpage-navigation.story.js

```diff
@@ -72,26 +74,37 @@ const btnIdHandler = () => {
   return false;
 };
 
+// Preserve the adapted specs.
+const prepareNavigation = data => {
+  // Buttons for the demo.
+  button(btnLeftLabel, btnLeftHandler, buttonLabels.cases);
+  button(btnMainLabel, btnMainHandler, buttonLabels.cases);
+  button(btnIdLabel, btnIdHandler, buttonLabels.cases);
+  button(btnIdRemoveLabel, btnIdRemoveHandler, buttonLabels.cases);
+
+  data.title = text('Title', data.title, buttonLabels.required);
+  data.links = object('Links', data.links, buttonLabels.required);
```

Comment:
here it should be
`links`
but no, ok, this was my idea but like these, is wrong, we pollute the data to build the demo, but this should be:
```
- "links" (array) (default: predefined structure): format: [
      {
        href: (string) (default: ''),
        label: (string) (default: '')
      },
      ...
      ]
```
So what we should do is to create two text knobs for each link that would be like:

`links[${i}].href` 
and
`links[${i}].label`

but you have to see what happens than if you use the buttons, to add or remove items from the inpage, i guess that the knobs would not be updated, but still it's acceptable.

Reply:
done @planctus, yes knobs are not updated if we play with the buttons to add or remove items.
This can be confusing in the end, I think..  maybe we should remove these buttons, what do you think?